requirements.txt

```diff
@@ -26,6 +26,7 @@ prettytable
 pyinotify<=0.10,>=0.9.5
 pymongo==3.2.2
 python-dateutil
+python-editor
```

Comment:
This file should get auto-generated during some make step and the version should be pinned as long as we put a pinned version into fixed-requierments.txt